app/frontend/src/components/EditLocationMap.tsx

```diff
@@ -55,136 +74,251 @@ export default function EditLocationMap({
   const [error, setError] = useState("");
 
   const map = useRef<mapboxgl.Map | null>(null);
-  //map is imperative so these don't need to cause re-render
+
+  // reactive location
+  const [rLocation, setRLocation] = useState<ApproximateLocation>(
+    location ?? {
+      address: "",
+      lat: 0,
+      lng: 0,
+      radius: 250,
+    }
+  );
+
+  // map is imperative so these don't need to cause re-render
   const centerCoords = useRef<LngLat | null>(
-    // TODO: better default?
-    location
-      ? new LngLat(location.lng, location.lat)
-      : new LngLat(151.2099, -33.865143)
+    location ? new LngLat(rLocation.lng, rLocation.lat) : new LngLat(35, 10)
```

Comment:
```suggestion
    new LngLat(rLocation.lng, rLocation.lat)
```
if location is null, the default values are used above anyway (change them to (35, 10))

Reply:
It doesn't sound right, but I was thinking if the unedited location is (0,0) then the backend might catch some issues with that. (35,10) is just to make the map look nice

Reply:
Can we instead not display the circle until they search for something?